lib/Elevate/Database.pm

```diff
@@ -15,10 +15,22 @@ use cPstrict;
 use Elevate::OS        ();
 use Elevate::StageFile ();
 
-use Cpanel::Pkgr ();
+use Cpanel::MysqlUtils::Version  ();
+use Cpanel::MysqlUtils::Versions ();
+use Cpanel::Pkgr                 ();
+
+use Log::Log4perl qw(:easy);
 
 use constant MYSQL_BIN => '/usr/sbin/mysqld';
 
+use constant SUPPORTED_MYSQL_VERSIONS => qw{
```

Comment:
QUESTION:

Should we update this name to be `SUPPORTED_CPANEL_MYSQL_VERSIONS` since CL does its own thing?

Reply:
Good idea.  I'll do that.